src/main/java/com/lob/client/AsyncLobClient.java

```diff
@@ -532,12 +532,16 @@ public void onThrowable(final Throwable t) {
         public T onCompleted(final Response response) throws Exception {
             if (isSuccess(response)) {
                 final T value = MAPPER.readValue(response.getResponseBody(), clazz);
```

Comment:
Unless I'm missing something about the FutureConverter. Is it ever used for anything besides just `
`AbstractResponse`s?